### PR TITLE
Gh pages 7.13 test release branch creation Team PSD Manual Update

### PR DESCRIPTION
**Pull request by jamesmrollins**
- Added instructions for test_version_release branch creation.
- Added a Readme.md in the actions folder to explain what the actions do.
- Added images to support user instructions.
- Updated _bookdown.yml to include new subchapter listing in TeamPSD manual

### CODE REVIEW

File: 7.13_test_release_branch_creation.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+5. When the test_version_release.yml file instructions are updated, commit the changes.
+6. Navigate back to the branch navigation button.
+
+<img src = "https://github.com/lzim/teampsd/blob/gh-pages/images/nav_find_create_branch.png?raw=true">
+
+7. Type in the new version test release branch name and press enter. The action should fire when the new branch is created. This can be monitored by clicking on the Actions tab.
+- type in "gh-pages_test_version_release_x_to_x"
+- the "test_version" in the name will trigger the action.
+- "x_x" indicates the current release and the future release.
```
Comment (epearman) [inline 7.13_test_release_branch_creation.md:38]: Do what we enter for x_x need to match any of the code in the yaml file for it to work or is what we enter for x_x arbitrary and for naming convention/tracking purposes only.

**Review by epearman — CHANGES_REQUESTED** [review]
@jamesmrollins, @emilymetcalf114 and I have reviewed these directions and made some change requests. We have not yet tested the actual function of the test branch yml because we are still a little unclear on some of the directions as we noted here.

**Review by ljmoody — COMMENTED** [review]
@epearman @emilymetcalf114 - Please see my comments and suggested changes based on your feedback and let me know your thoughts. 
FYI @jamesmrollins.

**Review by ljmoody — COMMENTED** [review]
@epearman @emilymetcalf114 - Please see my comments and suggested changes based on your feedback and let me know your thoughts. 
FYI @jamesmrollins.

**Review by epearman — APPROVED** [review]

**Pull request merged**